hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/protocol/SecretKeyProtocolServerSideTranslatorPB.java

```diff
@@ -102,6 +108,16 @@ public SCMSecretKeyResponse processRequest(SCMSecretKeyRequest request)
             .setSecretKeysListResponseProto(getAllSecretKeys())
             .build();
 
+      case GetCheckAndRotate:
+        try {
+          return scmSecurityResponse
+              .setCheckAndRotateResponseProto(
+                  checkAndRotate(request.getCheckAndRotateRequest().getForce()))
+              .build();
+        } catch (TimeoutException e) {
+          e.printStackTrace();
```

Comment:
LOG.error instead

Reply:
+1 error should be handled by the client (CLI)